src/sysmem.h

```diff
@@ -66,6 +66,19 @@ extern Int SyStorMin;
 /****************************************************************************
 **
 *V  SyAllocPool
+**
+**  'SyAllocPool' is the size of the OS memory block which Gasman is using
+**  to store its workspace.
+**
+**  Gasman's workspace must be a single continuous block, and can only be
+**  extended. Therefore extending this after GAP has been executing requires
+**  the OS does not allocate any memory immediately after the existing
+**  workspace. On 64-bit systems sing mmap this is usually possible as
```

Comment:
Is 'sing' meant to be 'using' here? Otherwise I'm not sure what it means.

Reply:
yup, thanks